packages/core/src/command/patient-import/commands/validate-and-parse-import.ts

```diff
@@ -38,38 +37,11 @@ export async function validateAndParsePatientImportCsv({
   const key = createFileKeyFiles(cxId, jobId, "raw");
   try {
     const csvAsString = await s3Utils.getFileContentsAsString(s3BucketName, key);
-    const allRows = csvAsString.split("\n");
-    const headersRow = allRows[0];
-    if (!headersRow) throw new Error(`File is empty for ${key}`);
-    const headers = normalizeHeaders(stripEol(headersRow).split(","));
-    if (!compareCsvHeaders(PatientImportCsvHeaders, headers)) {
-      throw new Error(`Headers are invalid for ${key}`);
-    }
-    const rows = allRows.slice(1);
-    if (rows.length === 0) throw new Error(`File is empty except for headers for ${key}`);
-    const validRows: string[][] = [];
-    const invalidRows: rowError[] = [];
-    const patients = rows.flatMap((row, rowIndex) => {
-      const rowColumns = stripEol(row).split(",");
-      if (rowColumns.length !== headers.length) {
-        invalidRows.push({
-          rowColumns,
-          error: `Row ${rowIndex} did not split into correct number of columns`,
-        });
-        return [];
-      }
-      const patientObject = createObjectFromCsv({ rowColumns, headers });
-      const parsedPatient = patientImportPatientSchema.safeParse(patientObject);
-      if (!parsedPatient.success) {
-        invalidRows.push({
-          rowColumns,
-          error: `Row ${rowIndex} had zod error ${JSON.stringify(parsedPatient.error)}`,
-        });
-        return [];
-      }
-      validRows.push(rowColumns);
-      return parsedPatient.data;
+
+    const { patients, invalidRows, validRows, headers } = await validateAndParsePatientImportCsv({
+      contents: csvAsString,
     });
+    console.log(invalidRows);
```

Comment:
Can we remove this?

Reply:
I think we want the files to be created before we continue no? Otherwise we're flying bling in my mind.

Reply:
Ah, this is from you no? I thought you were referring to the Promise.all for generating the files.

Reply:
No, this came from your PR (it shows as new line b/c I refactored the wrapping function.

Reply:
Yes we can remove (thought I removed it already) -- it's not on my current branch.